### DISCUSSION TITLE
Does this work with React-Native-Windows?

### DISCUSSION THREAD

**Discussion by dev1-akkess** [react-native-tvos/react-native-tvos#592] [Q&A] [answered]
As far as I can tell, [React Native Windows](https://github.com/microsoft/react-native-windows) is an extension to the base React Native package.  Does that mean I can use it together with this, to support iOS/Android/tvOS/AndroidTV/Windows all at once?

Thanks!

**Comment by douglowder** [comment, marked as answer]
@zap-dev1 I have not actually tried that. Discussing this with the RNWindows maintainers, it seems that it should be possible if using a RNWindows version that matches the RNTV version. I will investigate this (and also the same for RN MacOS).
